pyrokinetics/templates/input.gs2

```diff
@@ -63,7 +66,7 @@
 /
 
 &collisions_knobs
-    collision_model = 'default'
+    collision_model = 'none'
```

Comment:
Why do you need a collisionless case?

Reply:
@arkabokshi in case you don't receive notifications.

Reply:
We should try and match the collision operators as close as possible and I think including it is best. pyro doesn't currently change this value yet so we should leave it on by default

Reply:
OK. If we are inclined to keep as much physics as possible then we should enable collisions and beta_prime.  I think beta_prime makes it trickier to match; let me try again.

Reply:
Just use pyro to convert it